src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./Navbar.scss";
+import { Link } from "react-router-dom";
+
+const Navbar = () => {
+  return (
+    <nav className="navbar">
+      <ul className="filter-list">
+        <li className="filter-item">
+          <Link className="navbar-item" to="/Add">
+            Add
+          </Link>
+        </li>
+        <li className="filter-item">
+          <Link className="navbar-item" to="/">
```

Comment:
`to` value should be lower case as it's defined in the router.

Reply:
It is a / and it does not have lower or upper case. Am I true?

Reply:
Correct. My bad 😅